test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -893,9 +894,9 @@
       "Microsoft.Azure.Functions.JavaWorker/2.14.0": {},
       "Microsoft.Azure.Functions.NodeJsWorker/3.10.0": {},
       "Microsoft.Azure.Functions.PowerShellWorker.PS7.0/4.0.3148": {},
-      "Microsoft.Azure.Functions.PowerShellWorker.PS7.2/4.0.3131": {},
-      "Microsoft.Azure.Functions.PowerShellWorker.PS7.4/4.0.3147": {},
-      "Microsoft.Azure.Functions.PythonWorker/4.28.0": {},
+      "Microsoft.Azure.Functions.PowerShellWorker.PS7.2/4.0.3220": {},
```

Comment:
Looks like PR has more changes than update to Microsoft.Identity.Client package. Did you mean to include them as well?

Reply:
Yes. These are changes that already exist in the dev branch. deps.json was just not regenerated after some worker updates.